web/annotation_layer_builder.css

```diff
@@ -122,24 +132,33 @@
 .annotationLayer .choiceWidgetAnnotation select:hover,
 .annotationLayer .buttonWidgetAnnotation.checkBox input:hover,
 .annotationLayer .buttonWidgetAnnotation.radioButton input:hover {
-  border: 1px solid rgba(0, 0, 0, 1);
+  border: 2px solid var(--input-hover-border-color);
 }
 
 .annotationLayer .textWidgetAnnotation input:focus,
 .annotationLayer .textWidgetAnnotation textarea:focus,
 .annotationLayer .choiceWidgetAnnotation select:focus {
   background: none;
-  border: 1px solid transparent;
+  border: 2px solid var(--inner-focus-line-color);
+  border-radius: 2px;
+  outline: 1px solid Canvas;
```

Comment:
Here, and elsewhere below in the file, is it actually correct to add an `outline` *unconditionally* or was this intended only for HCM mode?
Assuming it was intended, does this interact "correctly" with the pre-existing `:required`-`outline` that's set in the `@media (forced-colors: active) {`-block above?  

In any case, should this perhaps be replaced with a variable rather than being repeated verbatim multiple times?

Reply:
Yes it's intended.
The lines you mentioned are only in the context of a focused field when the ones in forced-colors are for unfocused ones.
So I think it makes sense like this.

Reply:
As mentioned previously, can we please add a `--input-focus-outline: 1px solid Canvas;` variable and change all of `outline: 1px solid Canvas;`-lines to use that variable to avoid some duplication and simplify any future changes?